src/NVMLIOGroup.cpp

```diff
@@ -0,0 +1,673 @@
+                              {"NVML::PERFORMANCE_STATE", {
+                                  "Accelerator performance state",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::PCIE_RX_THROUGHPUT", {
+                                  "Accelerator PCIE receive throughput in bytes per second over a 20 millisecond period",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::PCIE_TX_THROUGHPUT", {
+                                  "Accelerator PCIE transmit throughput in bytes per second over a 20 millisecond period",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::CPU_ACCELERATOR_ACTIVE_AFFINITIZATION", {
+                                  "Returns the associated accelerator for a given CPU as determined by running processes."
+                                  "\n  If no accelerators map to the CPU then NAN is returned"
+                                  "\n  If multiple accelerators map to the CPU -1 is returned",
+                                  {},
+                                  GEOPM_DOMAIN_CPU,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::UTILIZATION_MEMORY", {
+                                  "Percentage of time the accelerator memory was accessed in the last set of driver samples",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::max,
+                                  string_format_double
+                                  }}
+                             })
+        , m_control_available({{"NVML::FREQUENCY_CONTROL", {
+                                    "Sets streaming multiprocessor frequency min and max to the same limit",
+                                    {},
+                                    GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                    Agg::average,
+                                    string_format_double
+                                    }},
+                               {"NVML::FREQUENCY_RESET_CONTROL", {
+                                    "Resets streaming multiprocessor frequency min and max limits to default values",
```

Comment:
How is the parameter used, if not used, please specify.  Perhaps it would be better to just enforce using a getter and a setter?

Reply:
It is not used, updated to reflect that.